x/concentrated-liquidity/tick.go

```diff
@@ -13,13 +13,10 @@ import (
 )
 
 // tickToSqrtPrice takes the tick index and returns the corresponding sqrt of the price
-func (k Keeper) tickToSqrtPrice(tickIndex sdk.Int) (sdk.Dec, error) {
-	price, err := sdk.NewDecWithPrec(10001, 4).Power(tickIndex.Uint64()).ApproxSqrt()
-	if err != nil {
-		return sdk.Dec{}, err
-	}
+func tickToSqrtPrice(tickIndex sdk.Int) sdk.Dec {
+	sqrtPrice := sdk.NewDecWithPrec(10001, 4).Power(tickIndex.Uint64() / 2)
```

Comment:
Q:

$$2^{3 /2} = 2^1.5 = 1.6817928305074292 $$

but with our `Power()` function it would be:

$$2^{3/2} = 2^1 = 2$$

Would it not? Am I missing anything?